Server/htdocs/AppController/commands_RSM/api/v2/items/update.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// ****************************************************************************************
+//Description:
+//  Edits one or more items of the specified itemTypeID with the associated values
+//
+//  REQUEST BODY (JSON)
+// Array with object/s inside, each object must contain
+//          - one propertyId and its value (one or more)
+//          - id: id of of the item being updated
+//  EXAMPLE:
+//      [{
+//          "109": "Roja"
+//          "ID": "0008"
+//        },{
+//          "ID":  "4001"
+//          "319": "Peter
+//          "320": "Parker"
+//        }]
+// ****************************************************************************************
+
+require_once "../../../utilities/RStools.php";
+require_once "../../../utilities/RSMverifyBody.php";
+setAuthorizationTokenOnGlobals();
+checkCorrectRequestMethod('PATCH');
+
+require_once "../../../utilities/RSdatabase.php";
+require_once "../../../utilities/RSMitemsManagement.php";
+require_once "../../api_headers.php";
+
+$requestBody = getRequestBody();
+verifyBodyContent($requestBody);
+
+// Definitions
+$clientID = getClientID();
+$RStoken =  getRStoken();
+$RSuserID =  getRSuserID();
+
+foreach ($requestBody as $item) {
+  $propertiesID = array();
+  //Iterate through every propertyID of the items to check if they are incongruent
+  foreach ($item as $propertyID => $propertyValue) {
+    if ($propertyID != "id" && $propertyID != "ID") {
+      $propertiesID[] = ParsePID($propertyID, $clientID);
+    }
+  }
+  $itemTypeIDID = getItemTypeIDFromProperties($propertiesID, $clientID);
+  $hasAllPermissions = checkTokenHasWritePermissions($RStoken, $RSuserID, $clientID, $propertiesID);
+  $itemID = $item->ID;
+  if ($itemTypeIDID == 0) {
+    $RSallowDebug ? returnJsonMessage(400, "Not Updated (Incongruent properties)")  : returnJsonMessage(400, "");
```

Comment:
I would add info here stating that properties must pertain to the same item type instead of mentioning "incongruent"